### DISCUSSION TITLE
How can I reset arena after every match for uhc

### DISCUSSION THREAD

**Discussion by ABIGDachshunch** [diamond-rip/Eden#96] [General]
The block is always there in the arena

**Comment by HierarchyMan** [comment]
U selected arena min/max properly?
U setup build height?

This issue isn't a problem for me

**Reply by ABIGDachshunch** [reply]
yes.

**Reply by ABIGDachshunch** [reply]
I try with another Spigot already

**Reply by HierarchyMan** [reply]
Just tested again with fox spigot
Isn't an issue for me
Try using one of the supported spigots

**Comment by GoodestEnglish** [comment]
> The block is always there in the arena

The issue should be fixed in the latest update. And it is confirmed to be work.
Try to use the latest version and reset the arena manually, then use `/save-all` to save the map, and restart the server. If it still doesn't work, send your arena.yml and tell us which map you are using.

**Reply by ABIGDachshunch** [reply]
Desert:
    icon: |
      rO0ABXNyABpvcmcuYnVra2l0LnV0aWwuaW8uV3JhcHBlcvJQR+zxEm8FAgABTAADbWFwdAAPTGph
      dmEvdXRpbC9NYXA7eHBzcgA1Y29tLmdvb2dsZS5jb21tb24uY29sbGVjdC5JbW11dGFibGVNYXAk
      U2VyaWFsaXplZEZvcm0AAAAAAAAAAAIAAlsABGtleXN0ABNbTGphdmEvbGFuZy9PYmplY3Q7WwAG
      dmFsdWVzcQB+AAR4cHVyABNbTGphdmEubGFuZy5PYmplY3Q7kM5YnxBzKWwCAAB4cAAAAAJ0AAI9
      PXQABHR5cGV1cQB+AAYAAAACdAAeb3JnLmJ1a2tpdC5pbnZlbnRvcnkuSXRlbVN0YWNrdAAFR1JB
      U1M=
    y-limit: 0
    build-max: 264
    portal-protection-radius: 3
    kits:
    - AxeUHC
    - Builduhc
    - Hypixeluhc
    - Archer
    enabled: true
    details:
      '0':
        a: '#w;NodebuffArena:#x;-56.08425535038175:#y;105.0:#z;333.1363022654176:#p;1.8098252:#yaw;358.17935'
        b: '#w;NodebuffArena:#x;-55.63123011009061:#y;105.0:#z;419.024328884174:#p;-0.0075750365:#yaw;179.67342'
        min: '#w;NodebuffArena:#x;-92.0:#y;98.0:#z;436.0:#p;0.0:#yaw;0.0'
        max: '#w;NodebuffArena:#x;-14.0:#y;150.0:#z;320.0:#p;0.0:#yaw;0.0'

**Comment by ABIGDachshunch** [comment]
I did all that my min max was set right and I redo thể map and the block still don't reset.. I set build max too

**Reply by GoodestEnglish** [reply]
Send a video where the bug happens, from server startup to the bug happens. The video should contain a coordinate mod. Currently I cannot reproduce the error.

**Comment by Shzv** [comment]
I just tested everything works fine for me
using papermc 1.8. maybe it's some fork that isn't supported or you just set the map wrong
 
edit:
try to put the build max at the limit of the builds you want to place for example your map is at a height of 100
you want to put the limedite of 5 blocks after which you can't build put as buildmax 105 and not by chance. I personally do this and I have no problems
